ksqldb-functional-tests/src/test/resources/rest-query-validation-tests/insert-values.json

```diff
@@ -332,6 +332,18 @@
       "outputs": [
         {"topic": "test_topic", "key": 1, "value": {"V0": {"F0": null}}}
       ]
+    },
+    {
+      "name": "should insert decimals that equal zero",
```

Comment:
we should also add a unit test in `DecimalUtilTest` - follow the pattern that is already there